jobs/async-upload/tests/test_models.py

```diff
@@ -0,0 +1,273 @@
+"""
+Unit tests for models.py - specifically testing model validation logic.
+"""
+import pytest
+from pydantic import ValidationError
+
+from job.models import ModelConfig, UploadIntent
+
+
+class TestModelConfigValidateModelIds:
+    """Test cases for ModelConfig.validate_model_ids method"""
+
+    def test_create_model_intent_with_all_ids_succeeds(self):
+        """Test that create_model intent succeeds when all required IDs are provided"""
+        config = ModelConfig(
+            upload_intent=UploadIntent.create_model,
+            id="test-model-id",
+            version_id="test-version-id",
+            artifact_id="test-artifact-id"
```

Comment:
not sure I got the intent of this test; if it's meant to create the triplet, why the triplet IDs are submitted and checked down?